SingularityService/src/main/java/com/hubspot/singularity/mesos/SingularitySlaveAndRackManager.java

```diff
@@ -115,6 +117,8 @@ SlaveMatchState doesOfferMatch(SingularityOfferHolder offerHolder, SingularityTa
 
     if (!isSlaveAttributesMatch(offerHolder, taskRequest, isPreemptibleTask)) {
       return SlaveMatchState.SLAVE_ATTRIBUTES_DO_NOT_MATCH;
+    } else if (!areSlaveAttributeMinimumsFeasible(offerHolder, taskRequest, activeTaskIdsForRequest)) {
+      return SlaveMatchState.SLAVE_ATTRIBUTES_DO_NOT_MATCH; // TODO: other enum?
```

Comment:
SLAVE_ATTRIBUTES_DO_NOT_MATCH enum is probably fine here, since it is still another case of this that we are checking for